decorator_allowed_groups/readme/USAGE.rst

```diff
@@ -0,0 +1,86 @@
+Once installed, the following code:
+
+.. code-block:: xml
+
+    <button type="object" name="my_action" groups="purchase.group_purchase_manager"/>
+
+.. code-block:: python
+
+    def my_action(self):
+        if not self.env.user.has_group("purchase.group_purchase_manager"):
+            raise AccessError(_("Some Error"))
+        pass
+
+can be replaced by:
+
+.. code-block:: xml
+
+    <button type="object" name="my_action"/>
+
+.. code-block:: python
+
+    @api.allowed_groups("purchase.group_purchase_manager")
```

Comment:
@legalsylvain IMO it's not a good practice to import the decorator from `odoo.api` this will only works if you add the addon into the `server_wide_modules` list and will require to modify the configuration file each time this addon is used to avoid strange error at server start. (Could become a pain in OCA repos). ` from odoo.addons.decorator.allowed_groups.api import allowed_groups` will work in every case without requiring to add the addon into the `server_wide_modules` list